### ISSUE TITLE
Colonists Not Working At All

### ISSUE THREAD

**Issue by AisuTistoto223** [ldtteam/minecolonies#3405]
### Minecolonies version

Version: 1.12.2-0.10.109-RELEASE-universal.jar 

### Expected behavior

Workers Do their Job

### Actual behaviour

Workers roam aimlessly as if they were on break.

### Steps to reproduce the problem

I'm unsure but it only happened after I installed the wither skeleton tweaks mod, started the world, then after it turned night set time to 0. I tried removing the new mod but when I did the world wouldn't load at all.

Please help. :(

**Comment by Graepix** [comment]
Don't quote me on this, but I'm pretty sure Colonist AI is heavily tied to the daylight cycle, meaning they break if you use change the time using commands.

**Comment by AisuTistoto223** [comment]
I can understand that but it's just never happened before. Maybe sometimes it does and sometimes it doesn't?

**Comment by Raycoms** [comment]
Did someone die? Might they be mourning?

Am Di., 19. Feb. 2019 um 20:49 Uhr schrieb AisuTistoto223 <
notifications@github.com>:

**Comment by AisuTistoto223** [comment]
I actually didn't think of that one. that makes sense. thanks.
But Now I have a new issue. I updated to the latest alpha and now I can't fulfill requests. at all. not with the item not in creative and not even with the item in creative. except for the builder and that only works with some items. I tried to go back to the release, but because items were added between that and the alpha, the world refuses to load without them for some reason.
